corehq/toggles/__init__.py

```diff
@@ -2494,6 +2486,18 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     help_link="https://confluence.dimagi.com/display/saas/Viewing+App+Changes+between+versions",
 )
 
+
+CASE_LIST_EXPLORER = FrozenPrivilegeToggle(
+    privileges.CASE_LIST_EXPLORER,
+    'case_list_explorer',
+    label='Show the case list explorer report',
+    tag=TAG_SOLUTIONS_OPEN,
+    namespaces=[NAMESPACE_DOMAIN],
+    save_fn=_ensure_search_index_is_enabled,
```

Comment:
Is the plan still to roll out the case search index to all domains?  Or only those with this privilege?

Reply:
only those with privilege or frozen FF